fenix/app/src/main/java/org/mozilla/fenix/compose/LinkText.kt

```diff
@@ -54,6 +56,8 @@ fun LinkText(
         textAlign = TextAlign.Center,
         color = FirefoxTheme.colors.textSecondary,
     ),
+    linkTextColor: Color = FirefoxTheme.colors.textAccent,
+    linkTextDecoration: TextDecoration? = null,
```

Comment:
This one makes sense, I'll make this change.  I think it makes a lot of sense to keep all the properties of the link-click portion of the text under the `LinkTextState`.

Reply:
👍

Reply:
actually this will require a more substantial rework than what I first thought. This might be better saved for a future rework if we end up touching `LinkText` again.
I'll reference this PR in the description so this conversation can be referenced if this gets picked up again.

Reply:
Created [follow-up ticket](https://bugzilla.mozilla.org/show_bug.cgi?id=1849256)